<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YUView</class>
 <widget class="QMainWindow" name="YUView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1262</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YUView</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_BufferInfo">
        <property name="title">
         <string>Buffer Info</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Pitch</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBox_BufferP">
           <property name="maximum">
            <number>8192</number>
           </property>
           <property name="value">
            <number>1920</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Height</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QSpinBox" name="spinBox_BufferH">
           <property name="maximum">
            <number>8192</number>
           </property>
           <property name="value">
            <number>1080</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_ImageInfo">
        <property name="title">
         <string>Image Info</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>x offset:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBox_OffsetX">
           <property name="maximum">
            <number>8192</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Y Offset:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QSpinBox" name="spinBox_OffsetY">
           <property name="maximum">
            <number>8192</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Width:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBox_ImageW">
           <property name="maximum">
            <number>8192</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Heigth:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QSpinBox" name="spinBox_ImageH">
           <property name="maximum">
            <number>8192</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_ImageInfo_2">
        <property name="title">
         <string>Pixel Info</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="3">
          <widget class="QPushButton" name="pushButton_SetPixel_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>More</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>x:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Y:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QSpinBox" name="spinBox_U">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="singleStep">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>V:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBox_PixelX">
           <property name="maximum">
            <number>8192</number>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QSpinBox" name="spinBox_V">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QPushButton" name="pushButton_SetPixel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QSpinBox" name="spinBox_PixelY">
           <property name="maximum">
            <number>8192</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>U:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBox_Y">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_26">
           <property name="text">
            <string>Y:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="ColorWidget" name="widget_Color" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>128</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>CFA</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4">
          <item>
           <widget class="QSplitter" name="splitter">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QListWidget" name="listWidget_CFA"/>
            <widget class="QGroupBox" name="groupBox_ImageInfo_3">
             <property name="title">
              <string>CFA Info</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QLabel" name="label_19">
                <property name="text">
                 <string>Pattern</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="comboBox_Compo_4">
                <item>
                 <property name="text">
                  <string>RGGB</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>GRBG</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>Demosaic</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="comboBox_Compo_3">
                <item>
                 <property name="text">
                  <string>Bylinear</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>DDFAPD</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="MonitorWidget" name="widget_MonitorCFA" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>YUV</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
          <item>
           <widget class="QSplitter" name="splitter_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QListWidget" name="listWidget_YUV"/>
            <widget class="QGroupBox" name="groupBox_ImageInfo_4">
             <property name="title">
              <string>CFA Info</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>Pattern</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_21">
                <property name="text">
                 <string>Demosaic</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="comboBox_Compo_8">
                <item>
                 <property name="text">
                  <string>IYUV</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>YV12</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>NV12</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>YU16</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>YV16</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>NV16</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="comboBox_Compo_2">
                <item>
                 <property name="text">
                  <string>Y'Cb'Cr</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Y only</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Cb only</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Cr only</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="MonitorWidget" name="widget_MonitorCFA_2" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1262</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_OpenFile"/>
    <addaction name="action_Save"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_OpenFile">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorWidget</class>
   <extends>QWidget</extends>
   <header>CustomWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MonitorWidget</class>
   <extends>QWidget</extends>
   <header location="global">CustomWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
